### PR TITLE
Added support for xml output

### PR DESCRIPTION
**Pull request by dragonito**

### CODE REVIEW

File: src/Formatter/Adapter/XmlAdapter.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+class XmlAdapter implements AdapterInterface
+{
+    /**
+     * @param Result $result
+     */
+    public function format(Result $result)
+    {
+        $serializer = new Serializer([new GetSetMethodNormalizer()], [new XmlEncoder()]);
+        echo $serializer->serialize($result, 'xml');
```
Comment (smoench) [inline src/Formatter/Adapter/XmlAdapter.php]: The result should be returned

File: src/Formatter/Adapter/XmlAdapter.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * @author Robin Willig <robin@dragonito.net>
+ */
+class XmlAdapter implements AdapterInterface
+{
+    /**
+     * @param Result $result
+     */
+    public function format(Result $result)
+    {
+        $serializer = new Serializer([new GetSetMethodNormalizer()], [new XmlEncoder()]);
```
Comment (smoench) [inline src/Formatter/Adapter/XmlAdapter.php]: Could you please use/extend the SerializerAdpater and introduce a new abstract method ```getEncoders()```

Comment (DavidBadura) [inline src/Formatter/Adapter/XmlAdapter.php]: no, you don't need a new method. you should add the XmlEncoder into the array in SerializerAdapter.

**Comment by dragonito** [comment]
updated

**Comment by DavidBadura** [comment]
:+1:

**Pull request merged**